### ISSUE TITLE
CLI: libkeytar error when dotcli login runs on github actions runners

### ISSUE THREAD

**Issue by dcolina** [dotCMS/core#28501]
Running the `dotCLI login` command on a GitHub Runner it has caught the following error:

```
[error]Exception in thread "main" java.lang.NoClassDefFoundError: Lcom/starxg/keytar/KeytarException;
       at com.oracle.svm.jni.functions.JNIFunctions.FindClass(JNIFunctions.java:348)
       at com.starxg.keytar.Keytar._setPassword(Keytar.java)
       at com.starxg.keytar.Keytar.setPassword(Keytar.java:86)
       at com.dotcms.api.client.KeyTarPasswordStoreImpl.setPassword(KeyTarPasswordStoreImpl.java:44)
       at com.dotcms.api.client.KeyTarPasswordStoreImpl_ClientProxy.setPassword(Unknown Source)
2024-05-08T18:10:41.8235335Z ##[error]
       at com.dotcms.api.client.HybridServiceManagerImpl.persist(HybridServiceManagerImpl.java:49)
       at com.dotcms.api.client.HybridServiceManagerImpl_ClientProxy.persist(Unknown Source)
       at com.dotcms.api.DefaultAuthenticationContextImpl.saveCredentials(DefaultAuthenticationContextImpl.java:140)
       at com.dotcms.api.DefaultAuthenticationContextImpl.login(DefaultAuthenticationContextImpl.java:120)
       at com.dotcms.api.DefaultAuthenticationContextImpl_ClientProxy.login(Unknown Source)
       at com.dotcms.cli.command.LoginCommand.call(LoginCommand.java:157)
       at com.dotcms.cli.command.LoginCommand_Subclass.call$$superforward1(Unknown Source)
       at com.dotcms.cli.command.LoginCommand_Subclass$$function$$6.apply(Unknown Source)
       at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:54)
       at io.quarkus.arc.impl.ActivateRequestContextInterceptor.aroundInvoke(ActivateRequestContextInterceptor.java:24)
       at io.quarkus.arc.impl.ActivateRequestContextInterceptor_Bean.intercept(Unknown Source)
       at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:41)
       at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:41)
       at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:32)
       at com.dotcms.cli.command.LoginCommand_Subclass.call(Unknown Source)
       at com.dotcms.cli.command.LoginCommand.call(LoginCommand.java:15)
       at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
       at picocli.CommandLine.access$1300(CommandLine.java:145)
       at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
       at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
       at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
       at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
       at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
       at com.dotcms.cli.common.DotExecutionStrategy.execute(DotExecutionStrategy.java:80)
       at io.quarkus.picocli.runtime.PicocliRunner$EventExecutionStrategy.execute(PicocliRunner.java:26)
       at picocli.CommandLine.execute(CommandLine.java:2078)
       at io.quarkus.picocli.runtime.PicocliRunner.run(PicocliRunner.java:40)
       at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:124)
       at io.quarkus.runtime.Quarkus.run(Quarkus.java:67)
       at io.quarkus.runtime.Quarkus.run(Quarkus.java:41)
       at io.quarkus.runner.GeneratedMain.main(Unknown Source)
```

Below, the runner specifications:

https://github.com/actions/runner-images/blob/ubuntu22/20240422.1/images/ubuntu/Ubuntu2204-Readme.md

**Comment by dcolina** [comment]
Internal QA: Passed.

After running several test cases, we can say: The bug is completely fixed.

Here is a test case built to check the fix.

https://github.com/dcolina/gha-course/actions/runs/9066707869/job/24910213110

At this PoC we have run a workflow with some steps to:

1. Login into demo site (automatically) through `dotCLI login` command. 
2. Then we've pulled the content with  `dotCLI pull` command. 
3. Then we've added a new file called dotcms-commands.vtl
4. Finally, we've pushed the file running `dotCLI push`

All those steps have been run through Github workflow.

Here is the final result on the web:

![Image](https://github.com/dotCMS/core/assets/29883881/656d83dd-c849-4ae2-a28e-232469916fbd)
